src/posts/datatree/index.md

```diff
@@ -0,0 +1,104 @@
+---
+title: 'Xarray x NASA: xarray.DataTree for hierarchical data structures'
+date: '2024-12-04'
+authors:
+  - name: Tom Nicholas
+    github: tomnicholas
+  - name: Owen Littlejohns
+    github: owenlittlejohns
+  - name: Matt Savoie
+    github: flamingbear
+  - name: Eni Awowale
+    github: eni-awowale
+  - name: Alfonso Ladino
+    github: aladinor
+  - name: Justus Magin
+    github: keewis
+  - name: Stephan Hoyer
+    github: shoyer
+summary: 'The new xarray.DataTree class allows working with netCDF/Zarr groups, brought to you in collaboration with NASA!'
+---
+
+## tl;dr
+
+[`xarray.DataTree`](https://docs.xarray.dev/en/stable/user-guide/data-structures.html#datatree) has been [released](https://github.com/pydata/xarray/discussions/9680) in [v2024.10.0](https://github.com/pydata/xarray/releases/tag/v2024.10.0), and the prototype [`xarray-contrib/datatree` repository](https://github.com/xarray-contrib/datatree) archived, after collaboration between the xarray team and the [NASA ESDIS project](https://www.earthdata.nasa.gov/about/esdis). 🤝
+
+## Why trees?
+
+Xarray users have been [asking](https://github.com/pydata/xarray/issues/4118) for a way to handle multiple netCDF4 groups [since at least 2016](https://github.com/pydata/xarray/issues/1092). Such netCDF4/Zarr groups are the on-disk representation of a general problem of handling hierarchies of related but non-alignable array data. Real-world datasets (such as [climate model intercomparisons](https://medium.com/pangeo/easy-ipcc-part-1-multi-model-datatree-469b87cf9114)) often fall into this category, and users wanted a way to work with such hierarchical data in-memory and a way to interact with it on disk.
+
+## What is a DataTree?
+
+Our solution is the new high-level container class `xarray.DataTree`.
+It acts like a tree of linked `xarray.Dataset` objects, with alignment enforced between variables in sibling nodes but not between parents and children. It can be written to and opened from formats containing multiple groups, such as netCDF4 files and Zarr stores.
```

Comment:
This description of the alignment requirement of DataTree looks backwards to me?
```suggestion
It acts like a tree of linked `xarray.Dataset` objects, with alignment enforced between parent and child nodes, but not between siblings. It can be written to and opened from formats containing multiple groups, such as netCDF4 files and Zarr stores.
```

Reply:
Oops you're right!